csm/core/agent/api.py

```diff
@@ -143,6 +143,21 @@ def http_request_to_log_string(request):
         method = request.method
         user_agent = request.headers.get('User-Agent')
         return (f"Remote_IP:{remote_ip} Url:{url} Method:{method} User-Agent:{user_agent}")
+    
+    @staticmethod
+    def process_audit_log(resp, request):
+        url = request.path
+        if (not request.app["usl_polling_log"]) \
+                and ("/usl/" in url) \
+                and (url not in ["/usl/v1/registerDevice"]):
```

Comment:
That condition could be better expressed as
```
if (not request.app['usl_polling_log')
        and url.startswith("/usl/")
        and url != '/usl/v1/registerDevice'):
    ...
```
What do you think?

The change of `not in` for a plain `!=` comparison is optional, but makes the code easier to read, IMO.

Reply:
@navalkp @tb-s  for this you guys can take decision and raise another PR. For now i am merging it.